src/table/utils/selections-utils.js

```diff
@@ -95,7 +104,17 @@ export function selectCell({ selectionState, cell, selectionDispatch, evt, annou
     return;
   }
 
-  selectedRows = getSelectedRows({ selectedRows, qElemNumber, rowIdx, evt });
+  if (isSelectMultiCells) {
```

Comment:
should be able to check what keys are pressed on `evt` and get rid of this argument